cob_hardware_config/cob4-1/config/image_flip_params_sensorring_cam3d.yaml

```diff
@@ -1,30 +1,22 @@
 # rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
 # int
-rotation_mode: 0
+rotation_mode: 1
```

Comment:
good point, but I fear we need it that way because if we bend torso and/or head, we'll rotate the camera freely in space and thus need to use the gravity method.